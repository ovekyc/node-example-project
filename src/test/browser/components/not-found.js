import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../../front/components/not-found';

it('renders correctly', ()=> {
  const tree = renderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
