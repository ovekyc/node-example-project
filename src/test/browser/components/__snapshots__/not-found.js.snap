// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    404 Not Found
  </h1>
</div>
`;
